Lib/fontTools/feaLib/parser.py

```diff
@@ -787,9 +792,11 @@ def parse_table_(self):
 
     def parse_table_GDEF_(self, table):
         statements = table.statements
-        while self.next_token_ != "}":
-            self.advance_lexer_()
-            if self.is_cur_keyword_("Attach"):
+        while self.next_token_ != "}" or (not self.ignore_comments and len(self.cur_comments_)):
```

Comment:
>  you can have any number of named parameters

only that you won't be able to pass those to the `next()` built-in function, which only takes the iterator object.